src/components/touchableOpacity/index.js

```diff
@@ -67,15 +69,25 @@ export default class TouchableOpacity extends BaseComponent {
   }
 
   render() {
-    const {throttle, ...others} = this.getThemeProps();
+    const {backgroundColor, borderRadius, paddings, margins, alignments, flexStyle} = this.state;
+    const {throttle, style, ...others} = this.getThemeProps();
 
     return (
       <RNTouchableOpacity
         {...others}
         onPress={this.onPress}
         onPressIn={this.onPressIn}
         onPressOut={this.onPressOut}
-        style={[this.props.style, this.backgroundStyle]}
+        style={[
+          backgroundColor && {backgroundColor},
+          borderRadius && {borderRadius},
```

Comment:
Why do you do backgroundColor && {backgroundColor}?

Reply:
I basically copied it from what I did in our `View` 
I guess the reason was not to assign something if it doesn't exist.

Reply:
oh. ok